docs/operator-manual/Rancher-Fleet.md

```diff
@@ -1,67 +1,67 @@
 ---
 sidebar_label: "Rancher Fleet"
-title: ""
+title: "Managing Kubewarden with Rancher Fleet"
+description: Managing Kubewarden with Rancher Fleet.
+keywords: [kubernetes, kubewerden, rancher fleet]
 ---
 
-# Managing Kubewarden with Rancher Fleet
+You can manage Kubewarden Helm charts,
+like other Helm charts, with [Rancher Fleet](https://fleet.rancher.io/).
+Rancher Fleet uses Kubernetes CRDs
+to define a GitOps approach to managing Kubernetes clusters.
+It does this by defining [Fleet Bundles](https://fleet.rancher.io/concepts).
 
-The Kubewarden Helm charts, like other Helm charts, can be managed via [Rancher
-Fleet](https://fleet.rancher.io/). Rancher Fleet uses Kubernetes CRDs to define
-a GitOps approach for managing Kubernetes clusters. It does this by defining
-Fleet Bundles.
+## Installing
 
-### Installing
+The Kubewarden charts are standard charts,
+they have dependencies (such as `cert-manager`),
+and depend transitively on each other
 
-The Kubewarden charts are normal charts, they have dependencies (such as
-`cert-manager`), and depend transitively on each other (`kubewarden-crds` <-
-`kubewarden-controller` <- `kubewarden-defaults`, see the [Quickstart
-docs](https://docs.kubewarden.io/quick-start)).
+`kubewarden-crds` ← `kubewarden-controller` ← `kubewarden-defaults`
 
-On Rancher Fleet, one can codify the chart dependencies using
-`dependsOn` in the [_fleet.yaml_ file](https://fleet.rancher.io/gitrepo-structure#fleetyaml).
+See the [Quickstart docs](https://docs.kubewarden.io/quick-start))
+for more information.
+
+Using Rancher Fleet, you can code the chart dependencies using
+`dependsOn` in the [`fleet.yaml`](https://fleet.rancher.io/ref-fleet-yaml) file.
+
+You may see transient errors until the charts are ready, such as:
 
-At the time of writing and by how Rancher Fleet works, one may see transient
-errors until the charts are ready, such as:
 ```
 ErrApplied(1) [Cluster fleet-local/local: dependent bundle(s) are not ready:
 [kubewarden-example-helm-kubewarden-controller]]
 ```
-These errors don't signify a problem, and once each chart has finished
-deploying, they will be gone.
 
-### Removing
+These errors don't signify a problem, and once the charts have finished deployment, they finish.
```

Comment:
I think this changes the meaning; this line was supposed to signify that the errors will not be there anymore.

Reply:
Thanks, I've adjusted that.